packages/react-menu/src/components/MenuList/MenuList.types.ts

```diff
@@ -1,7 +1,20 @@
 import * as React from 'react';
 import { ComponentProps } from '@fluentui/react-utils';
 
-export interface MenuListProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {}
+export interface MenuListProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  /**
+   * Callback when checked items change for value with a name
+   *
+   * @param name - the name of the value
+   * @param checkedItems - the items for this value that are checked
+   */
+  onCheckedValueChange?: (name: string, checkedItems: string[]) => void;
```

Comment:
We don't have a conclusion there, but let's add `event` as the first param anyway. Ideally it should match 1:1 to handlers on native elements

https://github.com/microsoft/fluentui/issues/12588

Reply:
Yes please, this general idea was actually something I thought about in the convergence sync but didn't have time to bring up: we definitely need to ensure that we have a convention established around change handler naming and arguments. I suspect we can agree that the event should always be the first arg, but we need someone to follow up on formalizing this and making sure everyone knows about it. @layershifter or @ling1726 (or feel free to nominate someone) would you be able to do that? I don't really have time right now and am probably not even the right person to do it since I haven't really been working on convergence.

Reply:
That makes sense to me, I'll change add the event now, and start translating #12588 to our current RFC style